test/e2e/coverage_docs/clusterpropagationpolicy_test.md

```diff
@@ -0,0 +1,30 @@
+| Test update resourceSelector of the propagation policy for the clusterRole | update resourceSelectors item(cluster scope)   |                                                                                                                             |
+| Test update propagateDeps of the propagation policy for deployment         | update policy propagateDeps(namespace scope)   |                                                                                                                             |
+| Test update placement of the propagation policy for deployment             | update policy placement(namespace scope)       |                                                                                                                             |
+| Test update placement of the propagation policy for clusterRole            | update policy placement(cluster scope)         |                                                                                                                             |
+
+#### ExplicitPriority propagation testing
+| Test Case                                                                        | E2E Describe Text                                                                         | Comments                                                                                                                                                                                             |
+|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Test the high explicit/low priority/implicit priority propagation for deployment | high explicit/low priority/implicit priority ClusterPropagationPolicy propagation testing | [Configure PropagationPolicy/ClusterPropagationPolicy priority](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#configure-propagationpolicyclusterpropagationpolicy-priority) |
+| Test the same explicit priority propagation for deployment                       | same explicit priority ClusterPropagationPolicy propagation testing                       | [Choose from same-priority PropagationPolicies](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#choose-from-same-priority-propagationpolicies)                                |
+
+#### Suggestions
+1. May need add the tase case when the [deployment updates](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#update-deployment).
+2. May need add the test case for the **same implicit priority** propagation.
+3. May need add the test case for **delete** the clusterPropagationPolicy.
```

Comment:
We can create an issue to track these tasks in the future.

Reply:
yes~ sure, and its work in progressnow i will add more